### PR TITLE
Issue#53 - Update Hydra's environment variables

### LINKED ISSUE #53: Update Hydra's environment variables
We are using deprecated envs for Hydra. We should update them to the newest configuration format. See [https://www.ory.sh/docs/hydra/configuration](https://www.ory.sh/docs/hydra/configuration)

### PR DESCRIPTION
**Pull request by claudiosegala**
Fix #53, start #12 and some minor improvements.

### CODE REVIEW

**Comment by claudiosegala** [comment]
@abilioesteves do you know any way to stress test every variable?

**Comment by abilioesteves** [comment]
@claudiosegala, what do you mean?

File: README.md [hunk 1]
```diff
@@ -87,13 +93,13 @@ From the project root folder, fire the following commands to execute this projec
         --shutdown-time    10 \
     ```
 
-4. Authorize application on hydra
-    
+5. Authorize application on hydra
```
Comment (abilioesteves) [inline README.md]: @claudiosegala, now that we have an example web app, couldn’t we just remove sections 5 and 6 and just add the example app as dependency at the available docker-compose.yml file?

Comment (claudiosegala) [reply README.md]: @abilioesteves, isn't it better to just add the whisper-examples link, instead of replacing?

**Comment by claudiosegala** [comment]
> @claudiosegala, what do you mean?

I mean that I change the variables and from what I saw, it seemed to work. But I do not think that the tests I made covered the usages of every variable that I changed.

**Comment by abilioesteves** [comment]
@claudiosegala, I think we should open a new issue to discuss how to test all the important variables (so Whisper can work nicely with Hydra). It's a complicated subject, since we are talking about integration testing (not unit testing). We should then discuss how to proceed with the creation of an Integration Testing pipeline, and then stress-out these variables.

File: README.md (hunk 1 above)
Comment (abilioesteves) [reply README.md]: The example we created is much better than the default one from Hydra. We'll have a docker image available for the example, making it easy for future users to execute Whisper and get a taste for its functionalities. The less cognitive overhead we have on our documentation, the better users will understand how Whisper works. Could you please do it, @claudiosegala? I'll start a docker image generation pipeline for the Web app whisper example.

**Comment by claudiosegala** [comment]
@abilioesteves, I agree with you. Should I open this issue?

**Comment by abilioesteves** [comment]
@claudiosegala, yes, please

**Pull request merged**